services/optimiser/calculator.py

```diff
@@ -74,11 +80,11 @@ def get_volunteer_by_index(self, index) -> User:
     def get_role_by_index(self, index) -> Role:
         return self._roles_[index]
 
-    def get_asset_request_by_index(self, index) -> AssetRequestVehicle:
-        return self._asset_request_vehicles_[index]
-
-    def get_asset_requests(self) -> List[AssetRequestVehicle]:
-        return self._asset_request_vehicles_
+    # def get_asset_request_by_index(self, index) -> AssetRequestVehicle:
```

Comment:
Please just delete these

Reply:
Just delete any commented out code you feel doesn't have a good reason to still be here (i.e. isn't pertaining to a feature that will be added back in the future)